### PR TITLE
Added Revert into hacktoberfest swag list issue - #830

### PR DESCRIPTION
**Pull request by nooras**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Closes: #830 

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -519,6 +519,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### R
 
+#### **Revert**
+
+- **Swag**: Stickers (100 pumpkins), Mug (250 pumpkins), Cap (250 pumpkins), Tshirt (500 pumpkins), Hoodie (1000 pumpkins)
```
Comment (vimode) [inline docs/list.md:524]:
```suggestion
- **Swag**: Stickers, Mug, Cap, T-shirt, and Hoodie.
```

File: docs/list.md [hunk 2]
```diff
@@ -519,6 +519,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Requirements**:
+    - Star our repo [here](https://github.com/revertinc/revert)
+    - Go to issues tagged Hacktoberfest
+    - For every PR merged in revert, you get spooky pumpkins
+    - Collect as many spooky pumpkins before 31st October 2023
+- **How to sign up**: No special signup known.
+- **Issues**: All the [open issues](https://github.com/revertinc/revert/issues)
+- **Notes**: More details [available here](hhttps://twitter.com/revertdotdev/status/1712834581383233988).
```
Comment (vimode) [inline docs/list.md:532]:
```suggestion
- **Notes**: More details [available here](https://twitter.com/revertdotdev/status/1712834581383233988).
```

File: docs/list.md [hunk 3]
```diff
@@ -519,6 +519,18 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+#### **Revert**
+
+- **Swag**: Stickers (100 pumpkins), Mug (250 pumpkins), Cap (250 pumpkins), Tshirt (500 pumpkins), Hoodie (1000 pumpkins)
+- **Requirements**:
+    - Star our repo [here](https://github.com/revertinc/revert)
+    - Go to issues tagged Hacktoberfest
+    - For every PR merged in revert, you get spooky pumpkins
+    - Collect as many spooky pumpkins before 31st October 2023
+- **How to sign up**: No special signup known.
+- **Issues**: All the [open issues](https://github.com/revertinc/revert/issues)
```
Comment (vimode) [inline docs/list.md:531]:
```suggestion
- **Issues**: All the open [hacktoberfest issues](https://github.com/revertinc/revert/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest).
```

**Review by vimode — CHANGES_REQUESTED** [review]
Thank you for contributing.

If these changes cause any code climate errors, can you please fix those too? Thank you.

**Comment by nooras** [comment]
> Thank you for contributing.
> 
> If these changes cause any code climate errors, can you please fix those too? Thank you.

Hi @vimode ,
Can you please explain a little more? I didn't get it !
Shall I just click on resolve conversation and commit the changes?
Shall I add pumpkins points in requirement?

**Comment by vimode** [comment]
Hey @nooras, the dark blue highlighted parts are the changes I did.
I removed the pumpkins points from the Swag field, its fine to keep them in requirements. And I also fixed the twitter link.

Yes you can click *commit the changes* to make those changes in this PR. 
Once you do that, this PR will get updated with those changes and all the checks will run again, codeclimate will do its thing to notify of any new errors.

**Comment by nooras** [comment]
Hi @vimode 
Have committed changes as you requested & also added 1 more point in requirements for that pumpkins points.
(I have made lots of commit 😅)
Kindly check and let me know if more changes are required.

Thanks for your suggestions

**Comment by crweiner** [comment]
Howdy @nooras, thank you for your PR to add Revert to the Hacktoberfest Swag List! And thank you too @vimode for the incredible assistance and for reviewing this PR! I've made some further tweaks, but it looks like things are now good to go, thanks again!

thumbs up

**Pull request merged**